Stack/Opc.Ua.Core/Types/Utils/Utils.cs

```diff
@@ -538,18 +538,31 @@ public static string ReplaceSpecialFolderNames(string input)
 
             StringBuilder buffer = new StringBuilder();
 
-            string value = Environment.GetEnvironmentVariable(folder);
-            if (value != null)
+#if NET46
```

Comment:
Please remove. The special folder should be configurable from the app config xml. We don't want ifdefs in the code.

Reply:
Environment.Specialfolders is now also supported in .Net 2.0, so I rather let it in, or the GDS client is broken Couldn't move the code to .Net46 layer because function is only called inside Opc.Ua.Core.